Source/WebCore/Modules/cookie-store/CookieStore.cpp

```diff
@@ -230,6 +231,20 @@ CookieStore::~CookieStore()
     m_mainThreadBridge->detach();
 }
 
+URL CookieStore::creationURL(ScriptExecutionContext& context) const
```

Comment:
Seems like it should be a static function in the cpp. It doesn't seem to rely on any class member?

Reply:
Fixed!